components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/exception/CertificateValidationManagementServerException.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.x509Certificate.validation.exception;
+
+/**
+ * X509 Configuration server exception.
+ */
+public class CertificateValidationManagementServerException extends CertificateValidationManagementException {
+
+    public CertificateValidationManagementServerException(String message) {
+
+        super(message);
+    }
+
+    public CertificateValidationManagementServerException(String message, Throwable cause) {
+
+        super(message, cause);
+    }
+
+    public CertificateValidationManagementServerException(String message, String description, String errorCode, Throwable cause) {
```

Comment:
Check the line limit

Reply:
fixed with https://github.com/wso2-extensions/identity-x509-commons/pull/57/commits/2f9c4cc0ce01a4d48b026d0659ff2194488b9c8a